installation/security.md

```diff
@@ -430,6 +431,149 @@ server {
     }
 }
 ```
+{: #synology-remote-config}
+#### Configuration on Synology DiskStation
+
+Synology DSM (as of 6.2) has the ability to automatically acquire certificates from Let's Encrypt and renew them every 90 days as required. The majority of the configuration mentioned above can be completed through the DSM GUI, but SSH access is required to implement authentication (**authentication is essential for remote access to your openHAB instance**).
```

Comment:
Please use one line per sentence.
The System will take care of a good presentation on the website.